sidebar/panel.js

```diff
@@ -214,6 +214,10 @@ port.onMessage.addListener((response) => {
       iframe.width = 300;
       iframe.setAttribute("src", '/sidebar/panelfeedback.html');
       break;
+    case 'MUTE':
+    case 'UNMUTE':
```

Comment:
Can we have this match with the 'shutup' constant that we are using inside of the foxycli?